src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -129,6 +129,13 @@ public override Conversion GetMethodGroupFunctionPointerConversion(BoundMethodGr
 
         protected override Conversion GetInterpolatedStringConversion(BoundExpression source, TypeSymbol destination, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo)
         {
+            if (_binder.InParameterDefaultValue || _binder.InAttributeArgument)
+            {
+                // We don't consider when we're in default parameter values or attributes to avoid cycles. This is an error scenario,
+                // so we don't care if we accidentally miss a parameter being applicable.
```

Comment:
>miss a parameter being applicable [](http://example.com/codeflow?start=55&length=33)

Is this comment correct?

Reply:
Yes. In the cases checked, we're checking parameter applicability, either in an attribute constructor parameter or the default value of a parameter.